docs/src/pages/components/badge.mdx

```diff
@@ -1,2 +1,152 @@
 # Badge
 
+The Badge component displays a dot or a circle with a count. It is normally used to indicate pending items (items in cart, unread messages, etc.)
+
+## Usage
+
+There are three sizes: `dot` (default) and `small` are meant to be used with an icon. `large` is meant to be used standalone.
+
+### Size dot with an icon
+```tsx
+<div style={{ padding: '2rem 1rem' }}>
+    <Badge
+        borderColor={'white'}
```

Comment:
You can't tell that the border or the hover color are happening because the background is already white. It's good to give someone a bit of prompting to mess around with the live code editor.

Reply:
My latest commit adds a section giving an example of the hover color in action 👍